js/src/data/resolvers.js

```diff
@@ -128,8 +127,25 @@ getGoogleMCContactInformation.shouldInvalidate = ( action ) => {
 	return action.type === TYPES.VERIFIED_MC_PHONE_NUMBER;
 };
 
-export function* getCountries() {
-	yield fetchCountries();
+export function* getMCCountriesAndContinents() {
+	try {
+		const query = { continents: true };
```

Comment:
There might not be a standard or convention, I was interpreting it from the `@wordpress/data` implementation design.

Actions are mainly the mechanism for making changes to states in the store (of wp-data), for example, a create/update/delete call corresponding to the API. And selectors are the primary mechanism for retrieving states from the store. Most states are obtained from APIs. So the reducers are the side effect handlers of the same name selectors, for example, a read call corresponding to the API.

Moreover, the UI often has multiple places requesting the same state in parallel, wp-data internally handles the same selector calls, allows the concurrent calls to issue only one API request, and notifies observers when the store is updated. In addition, its interface also includes the `isResolving`/`hasFinishedResolution` status of states.

If uses actions to retrieve data from API, it will lose the benefits of selector and resolver mechanisms. Also, from an external use point of view, the exposed state operation interfaces are selectors and actions when using wp-data. Moving this to resolvers.js could make the interface clearer and avoid misuse.

Reply:
Thanks, @eason9487 for your detailed explanation 😃